todo-angular-koa/app.js

```diff
@@ -84,7 +84,7 @@ function* update(next) {
             throw new Error("The todo must have a field `id`.");
         }
 
-        var result = yield r.table('todos').get(todo.id).update(todo, {returnVals: true}).run(this._rdbConn);
+        var result = yield r.table('todos').get(todo.id).update(todo, {returnChanges: true}).run(this._rdbConn);
         this.body = JSON.stringify(result.new_val);
```

Comment:
Sorry. I believe it should be `result.changes[0].new_val` -- but i'm not sure of the impact of only looking at `[0]`